### PR TITLE
[Add]: Note sending to archive and added side-bar

### PR DESCRIPTION
**Pull request by munjalSaksham71**
1.Added SideBar (NOTE: Designs are not finalize yet).
2.Get Notes only user added.
3.User can send note to archive and remove from archive too.

Issue: When user add/delete note: They have to reload . Fixing that in upcoming PR.

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is really good. i liked that you are exploring different technology

**Review by mariyasada — APPROVED** [review]
This code looks good to me. liked that you have done authentication for user

File: src/screens/HomeScreen.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.layout {
+    display: flex;
+    flex-flow: row wrap;
+    padding-top: 4rem;
+}
+
+.main {
+    margin-left: 8rem;
+    padding-top: 4rem;
+    padding-left: 3rem;
```
Comment (SaurabhChawda) [inline src/screens/HomeScreen.css:10]: You can use shorthand property.

**Comment by SaurabhChawda** [comment]
-everything looks good to me. you can merge it.

**Review by SaurabhChawda — APPROVED** [review]

**Pull request merged**